.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy til prod-sbs
+on:
+  workflow_run:
+    workflows: ["Build for prod"]    # Venter på at bygget for prod er ferdig
+    branches: [master]
```

Comment:
Hvis "Build for prod" og "Build" er så like at de kunne biltt slått sammen, kunne denne ventet på "Build" på master? 🤔

Reply:
Ja! Endrer til det i #207

Reply:
Så ikke den andre PRen før etter jeg kommenterte 😄

Reply:
Tenkte jeg skulle dele opp for å gjøre PR-ene mindre og mer oversiktelige, men tror kanskje det ble mer rotete sånn 😅